README.md

```diff
@@ -1,2 +1,82 @@
+_And what does this mean?_ Well, by implementing a high-performant consensus we have increased the throughput of the network significantly while reducing the 30 second block time consensus to around 1 second. As you may already be aware, Filecoin recently launched its [Filecoin Virtual Machine](https://fvm.filecoin.io/) and it will soon release support for user-defined actor. This will on board a great gamut of new applications to the Filecoin network significantly increasing its load. Certain use cases may realize that they need more from Filecoin in terms of throughput and finality times, and here is were Spacenet comes in.
+
+With Spacenet we want to provide developers with a testbed to deploy their FVM use cases and innovate with new Web3 applications. Many of you may be wondering, _but why would I want to develop my application over a high-throughput Filecoin network if my goal is to deploy it on mainnet?_ Well, Spacenet is just the first step towards the deployment of the [InterPlanetary Consensus (IPC)](https://github.com/filecoin-project/FIPs/discussions/419), and our ambitious plan of scaling Filecoin (in terms of performance and new capabilities). Now Spacenet is just Filecoin with a faster consensus algorithm, but in the future it will be much more. Spacenet is just your portal to glimpse what is yet to come to the Filecoin ecosystem (but we are getting ahead of ourselves). 
+
+Spacenet is not only a testbed for developer to test the new improvements to the protocol we are working on, it is also a way for us to test our consensus innovations with real applications and real users. For instance, once IPC is released in Spacenet, developers will be able to deploy their own subnets from Spacenet while maintaining the ability to seamlessly interact with state and applications in the original network from which they have become independent. With this version of Spacenet, we want to test Mir-Trantor, the first consensus algorithm supported by IPC subnets, before we release full support for IPC. 
+
+> In the meantime, to learn more about IPC you can read [this paper](https://research.protocol.ai/publications/hierarchical-consensus-a-horizontal-scaling-framework-for-blockchains/) and/or [watch this talk](https://www.youtube.com/watch?v=bD1LDVc2lMQ&list=PLhuBigpl7lqu0bsMQ8K7aLfmUFrkMw52K&index=3).
+
+## SLA of the network
+Spacenet is an experimental network. We'll do our best to have it always running, but some hiccups may appear along the way. If you are looking to rely on Spacenet for your applications you should expect:
+- Unexpected (and potentially long-lasting) downtimes while we investigate bugs.
+- Complete restarts of the network and loss of part or all stored state. If really nasty bugs appear we may need to restart the network from a previous checkpoint, or completely restart from genesis.
+- Bugs and rough edges to be fixed and polished along the way.
```

Comment:
```suggestion
Spacenet is an experimental network. We aim to have it constantly running, but some hiccups may appear along the way. If you are looking to rely on Spacenet for your applications, you should expect:
- Unplanned (and potentially long-lasting) downtime while we investigate bugs.
- Complete restarts of the network and loss of part or all stored state. In case of serious issues, it may be necessary to restart the network from a previous checkpoint, or completely restart from genesis.
- Bugs and rough edges to be fixed and polished along the way.
```